AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/ActivityWithServices.java

```diff
@@ -279,4 +279,28 @@
 	@SystemService
 	WindowManager windowManager;
 
+	AppWidgetManager methodInjectedAppWidgetManager;
```

Comment:
should they? in other tests similar fields are default too. thats why i left the visibiliy.